files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -1697,6 +1822,12 @@
     - [[ .Vault ]]:9005
     labels:
       app: cert-exporter
+[[- if or (eq .Installation "dragon") (eq .Installation "dinosaur") ]]
+  - targets: 
+    - [[ .Vault ]]:9101
+    labels:
+      app: haproxy-exporter
+[[- end ]]
```

Comment:
I do hate this condition, do we really need this? If yes please document why and/or link to some issue.